draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -270,6 +270,11 @@ Key material used for authentication in unprompted authentication, whether
 symmetric or asymmetric MUST NOT be reused in other protocols. Doing so can
 undermine the security guarantees of the authentication.
 
+Sites offering Unprompted Authentication are able to link requests that use
+the same key for the Authentication Schemes provided. However, requests are
+not linkable across other sites if the keys used are private to the individual
```

Comment:
"private" is not really the right word to use in this context.  I think that "unique" would be better, or you could use the opposite: provided that the keys are not used for requests to other sites.

Note that the term "site" probably needs to be defined here.  Origin is probably an easier term to lean on.